Podfile.lock

```diff
@@ -13,4 +13,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: f4462aedc7398f89b3216737d86da7f673eb4649
 
-COCOAPODS: 1.11.3
+COCOAPODS: 1.11.2
```

Comment:
Just to double check, any reason that we want to downgrade this?

Reply:
That's by accident, will undo this.